www/src/components/LayoutHeader/Header.js

```diff
@@ -23,8 +23,19 @@ import logoSvg from 'icons/logo.svg';
 // This is how the previous Jekyll site determined version though.
 const {version} = require('../../../../package.json');
 
+import {css} from 'glamor';
+
+css.global('#header', {
+  transition: 'transform 0.2s ease',
+});
+
+css.global('.is-menu-open #header', {
+  transform: 'translateY(-100%)',
+});
```

Comment:
It would be nice to avoid global styles for this. I suggest we pass an `isOpen` prop do this component instead and (probably) expose a toggle function to the sticky sidebar via context.